### ISSUE TITLE
Separate files for each activity

### ISSUE THREAD

**Issue by lukashino** [karlicoss/endoexport#6]
I think other sport application require activities to be divided in separate files. It would be nice to have there although as Endomondo is at the end I am not really expecting to be it implemented. It can possibly serve others. 
data.gpx is the file exported from your application
data[1-n].gpx are files of individual activities

```py
import json

with open('data.gpx') as json_file:
    data = json.load(json_file)
    i = 0
    for o in data:
        i+=1
        with open(f'data{i}.gpx', 'w', encoding='utf-8') as f:
            json.dump(o, f, ensure_ascii=False, indent=4)

```

**Comment by karlicoss** [comment]
Yeah probably, but to convert in GPX likely more work would be required anyway, as far as I know GPX is more like XML? Likely, time format is also different.
